css/style.css

```diff
@@ -17,3 +18,251 @@ body {
  */
 
 
+
+/* HEADER */
+
+.wrapper {
+    width: 1170px;
+    margin: 0 auto;
+}
+
+
+header{
+    height: 50px;
+    background: whitesmoke;
+    width: 100%;
+    position:relative;
+}
+
+.header-logo {
+    width: 40px;
+    height: 40px;
+    float:left;
+}
+
+nav {
+    float:right;
+    line-height: 30px;
+}
+
+nav a {
+    margin: 0 20px;
+    text-decoration: none;
+    color: rgb(129, 129, 129);
+    font-weight: 400;
+
+}
+
+nav a:nth-of-type(1) {color: black; font-weight: 500;}
+
+nav a:hover {color:chocolate;}
+
+
+
+/* MAIN (1st Section) */
+
+.banner-area {
+    position: relative;
+}
+
+
+
+
+
+.img {
```

Comment:
I made a code sample based on one of TA's work for making the background sitting on the full page:
https://codepen.io/mahsa2-the-bold/pen/WNEPyYO

We can talk more about it when we meet next time :)